content/episodes/0010-changing-bases-decades-of-the-web.md

```diff
@@ -13,41 +13,45 @@ explicit = "No"
 
 # Guest: Luke Williams
 - Bio
-- Website
+- [Website](http://red-root.com/)
 
 # Front End WebDev (pre-2003)
-- HTML
-- CSS
-- JavaScript
-- Browser incompatibilities
-- Flash
-- Shockwave
-- Java Applets
+- [HTML](https://en.wikipedia.org/wiki/HTML)
+- [CSS](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+- [JavaScript](https://en.wikipedia.org/wiki/JavaScript)
+- [Browser incompatibilities](https://www.keycdn.com/blog/browser-compatibility-testing-tools/)
```

Comment:
hard to find a good source for this as it's constantly in flux, decided to link to a list of tools for checking in different browsers instead.